core/src/test/scala/integration/kafka/api/ConsumerBounceTest.scala

```diff
@@ -104,6 +104,11 @@ class ConsumerBounceTest extends IntegrationTestHarness with Logging {
     consumer.subscribe(tp)
     consumer.seek(tp, 0)
 
+    // wait until all the followers have synced the last HW with leader
```

Comment:
@guozhangwang Perhaps I was mistaken, but I thought the issue was that the writes were getting lost before the first seekToEnd, which means that it fails to find the expected position. I don't think it was because of inconsistency in the ordering of offsets visible to the user.